config/environments/test.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Some tests require the following languages (en, es, fr, nl, pt-BR)
+  # so we override the available languages for the test environment.
+  config.i18n.default_locale = :en
```

Comment:
I think we can omit this line, since we set this value in `spec/rails_helper`: https://github.com/consul/consul/blob/master/spec/rails_helper.rb#L16

Reply:
Actually, we need this line because the test environment will fail to load if the default locale defined in `config/application.rb` isn't present in the available locales defined in the test environment.

So what do you think about removing the `I18n.default_locale = :en` line in `spec/rails_helper.rb`?